python/semantic_kernel/connectors/ai/google/vertex_ai/vertex_ai_settings.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from typing import ClassVar
+
+from semantic_kernel.kernel_pydantic import KernelBaseSettings
+
+
+class VertexAISettings(KernelBaseSettings):
+    """Vertex AI settings.
+
+    The settings are first loaded from environment variables with
+    the prefix 'VERTEX_AI_'.
+    If the environment variables are not found, the settings can
+    be loaded from a .env file with the encoding 'utf-8'.
+    If the settings are not found in the .env file, the settings
+    are ignored; however, validation will fail alerting that the
+    settings are missing.
+
+    Required settings for prefix 'VERTEX_AI_' are:
+    - gemini_model_id: str - The Gemini model ID for the Vertex AI service, i.e. gemini-1.5-pro
+                This value can be found in the Vertex AI service deployment.
+                (Env var VERTEX_AI_GEMINI_MODEL_ID)
+    - embedding_model_id: str - The embedding model ID for the Vertex AI service, i.e. text-embedding-004
+                This value can be found in the Vertex AI service deployment.
+                (Env var VERTEX_AI_EMBEDDING_MODEL_ID)
+    - project_id: str - The Google Cloud project ID.
+                (Env var VERTEX_AI_PROJECT_ID)
+    """
+
+    env_prefix: ClassVar[str] = "VERTEX_AI_"
```

Comment:
is `VERTEX_AI` enough of a prefix here? Do we want to include `GOOGLE_` to make it `GOOGLE_VERTEX_AI_`?

Reply:
This is Google's way of naming things: https://cloud.google.com/vertex-ai/generative-ai/docs/migrate/migrate-google-ai.

I think it's ok to have `Google AI` and `Vertex AI`. This aligns with what Google has. And we already have both under the module `connectors/ai/google` so I think it's clear enough.

Reply:
In terms of the settings, I think `VERTEX_AI_` is ok, unless we have naming conflicts.